backend/capellacollab/settings/modelsources/git/crud.py

```diff
@@ -23,29 +23,26 @@ def get_git_settings(db: Session) -> list[DatabaseGitInstance]:
     return db.query(DatabaseGitInstance).all()
 
 
-def create_git_setting(
+def create_git_instance(
```

Comment:
Agree with the point that `setting` is not the best term here but IMO such renaming should not be performed iteratively as it may cause some confusions and in addition, usually doesn't take that long. But anyways opened #603 which should remove all the remaining legacy names (except the database table which I can also add to the PR next week), so we can merge it in after this PR is done.

Reply:
Renamed the table already. Nice, didn't see your PR :)